source/js/buyers-guide/product-analytics.js

```diff
@@ -1,117 +1,128 @@
       action: `share tap`,
       label: `share site to Facebook`,
       transport: `beacon`
     },
-    'nav-social-button-twitter': {
+    '#nav-social-button-twitter': {
       category: `buyersguide`,
       action: `share tap`,
       label: `share site to Twitter`,
       transport: `beacon`
     },
-    'nav-social-button-email': {
+    '#nav-social-button-email': {
       category: `buyersguide`,
       action: `share tap`,
       label: `share site to Email`,
       transport: `beacon`
     },
-    'nav-social-button-fb-mb': {
+    '#nav-social-button-fb-mb': {
       category: `buyersguide`,
       action: `share tap`,
       label: `share site to Facebook`,
       transport: `beacon`
     },
-    'nav-social-button-twitter-mb': {
+    '#nav-social-button-twitter-mb': {
       category: `buyersguide`,
       action: `share tap`,
       label: `share site to Twitter`,
       transport: `beacon`
     },
-    'nav-social-button-email-mb': {
+    '#nav-social-button-email-mb': {
       category: `buyersguide`,
       action: `share tap`,
       label: `share site to Email`,
       transport: `beacon`
     },
+
     // product events
-    'product-copy-link-button': {
+    '#product-company-url': {
+      category: `product`,
+      action: `company link tap`,
+      label: `company link for ${productName}`,
+      transport: `beacon`
+    },
+    '#product-copy-link-button': {
       category: `product`,
       action: `copy link tap`,
       label: `copy link ${productName}`
     },
-    'creep-vote-btn': {
+    '#product-live-chat': {
+      category: `product`,
+      action: `customer support link tap`,
+      label: `support link for ${productName}`
+    },
+    '#creep-vote-btn': {
       category: `product`,
       action: `opinion submitted`,
       label: `opinion on ${productName}`
     },
-    'product-social-button-fb': {
+    '#product-social-button-fb': {
       category: `product`,
       action: `share tap`,
       label: `share ${productName} to Facebook`,
       transport: `beacon`
     },
-    'product-social-button-twitter': {
+    '#product-social-button-twitter': {
       category: `product`,
       action: `share tap`,
       label: `share ${productName} to Twitter`,
       transport: `beacon`
     },
-    'product-social-button-email': {
+    '#product-social-button-email': {
       category: `product`,
       action: `share tap`,
       label: `share ${productName} to Email`,
       transport: `beacon`
     },
-    'privacy-policy-link': {
+    '#privacy-policy-link': {
       category: `product`,
       action: `privacy policy link tap`,
       label: `policy link for ${productName}`,
       transport: `beacon`
     },
-    'reading-level-link': {
+    '#reading-level-link': {
       category: `product`,
       action: `carnegie mellon reading level links`,
       label: `reading level link for ${productName}`,
       transport: `beacon`
+    },
+
+    // product updates
+    'a.product-update-link': {
```

Comment:
one suggestion:

we can probably just have `.product-update-link` here?

Reply:
yep